js/RTCPeerConnection.js

```diff
@@ -584,6 +587,84 @@ RTCPeerConnection.prototype.createDataChannel = function (label, options) {
 	return new RTCDataChannel(this, label, options);
 };
 
+RTCPeerConnection.prototype.getStats = function () {
+	var self = this,
+		isPromise,
```

Comment:
This is inconsistent with the rest of the plugin. IMHO the switch to promises should happen all at once in a major release, say 4.0. So IMHO I'd just have the callback API here.